src/Jackett.Common/Definitions/yggtorrent.yml

```
@@ -8,7 +8,7 @@ encoding: UTF-8
 followredirect: true
 requestDelay: 2
 links:
-  - https://www6.yggtorrent.lol/
+  - https://www3.yggtorrent.do/
 legacylinks:
```

```suggestion
legacylinks:
  - https://www6.yggtorrent.lol/
```
in future please add old sitelink to the legacylinks so that automation can auto-update the users indexer configuration

Also, please update yggcookie the same way. thanks.